modules/jwplayerRtdProvider.js

```diff
@@ -71,15 +81,31 @@ export function fetchTargetingInformation(jwTargeting) {
   });
 }
 
+export function setOverrides(params) {
+  // For backwards compatibility, default to always unless overridden by Publisher.
+  overrideContentId = sanitizeOverrideParam(params.overrideContentId, ENRICH_ALWAYS);
```

Comment:
Assuming this should be updated for 9, do you have a way to track it - would a TODO comment be helpful?

Reply:
added!